client/service/src/builder.rs

```diff
@@ -198,6 +206,41 @@ pub type TLightClientWithBackend<TBl, TRtApi, TExecDisp, TBackend> = Client<
 	TRtApi,
 >;
 
+/// Construct and hold different layers of Keystore wrappers
+pub struct KeystoreContainer {
```

Comment:
I am a bit confused by this struct. Now that `CryptoStore` and `SyncCryptoStore` are traits with the latter having an auto trait implementation for the former, why do you ever need a reference to both a `CryptoStore` and a `SyncCryptoStore`?